CHANGES.md

```diff
@@ -1,3 +1,29 @@
+## 1.5.1 2018-10-24
+
+Linkerd 1.5.1 adds an exciting new interpreter to its suite of interpreter module. The new 
+`io.l5d.consul.interpreter` now allows Linkerd to read dtabs directly from a Consul KV store agent
+that This effectively removes the dependence on Namerd to resolve service names to concrete names
+and simplifies Linkerd's deployment model in Consul backed architectures. In addition, this release
+adds a number of critical bug fixes to the HTTP/2 router that fixes an issue where Linkerd would 
+sometimes degrade HTTP/2 streaming over some period of time.
+
+This release features contributions from OfferUp, Planet Labs and Buoyant with a special shoutout
+to [Leo Liang](https://github.com/leozc) and [Chris Taylor](https://github.com/ccmtaylor) for their
+work on fixing a bug in the DNS SRV namer.
+
+Full release notes:
+
+* HTTP/2
+  * Fixes an issue in HTTP/2 streaming that causes Linkerd to no longer route HTTP/2 frames after receiving a close frame from an upstream client.
+  * Adds support for setting the `maxConcurrentStreamsPerConnection` config value to 1000 by default to prevent Linkerd from running out of memory when HTTP/2 clients leak connection streams.
```

Comment:
I wouldn't say "Add support" because we already had support.  Maybe "Sets `max...` to 1000 by default..."